nilearn/plotting/data/html/stat_map_template.html

```diff
@@ -28,7 +28,7 @@
   // On load: build all figures
   $( window ).on('load',function() {
     // Create brain slices
-    var brain = brainsprite(
```

Comment:
Actually, it looks like there are a few tests checking for `var brain` in the html code, and which are now failing.

Reply:
Ah, I couldn't find them - that's probably because this a template, not an actual HTML document. I'll revert that.

Reply:
Reverted, although I haven't understood what exactly was using the variable.

Reply:
Some tests are doing some basic checks on the raw html output and look for the presence of the string "var brain" within the html string.